src/MockClassifier.Api/Controllers/InstitutionController.cs

```diff
@@ -1,12 +1,25 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+using MockClassifier.Api.Interfaces;
 using MockClassifier.Api.Models;
+using MockClassifier.Api.Services.Dmr;
 
 namespace MockClassifier.Api.Controllers
 {
     [Route("input-from-dmr/institution")]
     [ApiController]
     public class InstitutionController : ControllerBase
     {
+        private readonly IDmrService _dmrService;
+        private readonly ITokenService _tokenService;
+        private readonly INaturalLanguageService _naturalLanguageService;
+
+        public InstitutionController(IDmrService dmrService,ITokenService tokenService, INaturalLanguageService naturalLanguageService)
+        {
+            _dmrService = dmrService;
+            _tokenService = tokenService;
+            _naturalLanguageService = naturalLanguageService;
+        }
+
         /// <summary>
         /// Processes an array of strings to identify corresponding ministries and issues call backs to DMR for each message.
```

Comment:
I feel like has been mention somewhere that the message won't necessarily always be passed between ministries, but maybe that actually doesn't apply for the mock stuff yet, so might not be relevant.

Reply:
@joonasroosalung The purpose of this is not to pass anything directly to ministries but to receive a message and then respond back to DMR (via callback) with detail of which ministry the message has been resolved to so that DMR can pass the message along. Does that address the question?